### PR TITLE
Separate xvf3510 into INT and UA products and improve readme text

### LINKED ISSUE #43: Sensory repo not found
I'm trying to follow the instructions for install AVS.  When the script attempts to clone the Sensory repo, it fails, because there are no public Sensory repos.  Is there a workaround for this?

### PR DESCRIPTION
**Pull request by lucianomartin**
Fixes #43 #47 https://github.com/xmos/vocalfusion-avs-setup/issues/50 https://github.com/xmos/vocalfusion-avs-setup/issues/53 https://github.com/xmos/vocalfusion-avs-setup/issues/54

### CODE REVIEW

File: auto_install.sh [hunk 1]
```diff
@@ -7,12 +7,12 @@ RPI_SETUP_REPO=vocalfusion-rpi-setup
 RPI_SETUP_DIR=$SETUP_DIR/$RPI_SETUP_REPO
 RPI_SETUP_SCRIPT=$RPI_SETUP_DIR/setup.sh
 
-RPI_SETUP_TAG="v5.0.0"
-AVS_DEVICE_SDK_TAG="v1.25.0.2"
+RPI_SETUP_TAG="feature/3510_updates"
```
Comment (mbanth) [inline auto_install.sh:10]: It seems odd to use a feature branch here.  Are you planning to replace it once another pull request completes?

File: auto_install.sh [hunk 2]
```diff
@@ -7,12 +7,12 @@ RPI_SETUP_REPO=vocalfusion-rpi-setup
...
+AVS_DEVICE_SDK_TAG="feature/remove_sensory"
```
Comment (mbanth) [inline auto_install.sh:11]: It seems odd to use a feature branch here.  Are you planning to replace it once another pull request completes?

File: auto_install.sh [hunk 3]
```diff
@@ -131,7 +131,7 @@ if [ -d $RPI_SETUP_DIR ]; then
   rm -rf $RPI_SETUP_DIR
 fi
 
-git clone -b $RPI_SETUP_TAG https://github.com/xmos/$RPI_SETUP_REPO.git
+git clone -b $RPI_SETUP_TAG https://github.com/lucianomartin/$RPI_SETUP_REPO.git
```
Comment (mbanth) [inline auto_install.sh:134]: It seems odd to use a fork here.  Are you planning to replace it once another pull request completes?

File: auto_install.sh [hunk 4]
```diff
@@ -156,8 +156,8 @@ if $RPI_SETUP_SCRIPT $XMOS_DEVICE; then
   wget -O pi.sh https://raw.githubusercontent.com/xmos/avs-device-sdk/$AVS_DEVICE_SDK_TAG/tools/Install/pi.sh
   wget -O genConfig.sh https://raw.githubusercontent.com/xmos/avs-device-sdk/$AVS_DEVICE_SDK_TAG/tools/Install/genConfig.sh
   chmod +x $AVS_SCRIPT
-AVS_CMD="./${AVS_SCRIPT} ${CONFIG_JSON_FILE} ${AVS_DEVICE_SDK_TAG} -s ${DEVICE_SERIAL_NUMBER} -x ${XMOS_DEVICE} ${GPIO_KEY_WORD_DETECTOR_FLAG} ${HID_KEY_WORD_DETECTOR_FLAG}"
-echo "Running command ${AVS_CMD}"
+  AVS_CMD="./${AVS_SCRIPT} ${CONFIG_JSON_FILE} ${AVS_DEVICE_SDK_TAG} -s ${DEVICE_SERIAL_NUMBER} -x ${XMOS_DEVICE} ${GPIO_KEY_WORD_DETECTOR_FLAG} ${HID_KEY_WORD_DETECTOR_FLAG}"
```
Comment (mbanth) [inline auto_install.sh:159]: Why the leading white space?

File: auto_install.sh [hunk 5]
```diff
@@ -156,8 +156,8 @@ if $RPI_SETUP_SCRIPT $XMOS_DEVICE; then
...
+  echo "Running command ${AVS_CMD}"
```
Comment (mbanth) [inline auto_install.sh:160]: Why the leading white space?

File: auto_install.sh (hunk 4 above)
Comment (lucianomartin) [reply auto_install.sh:159]: They were missing, I just fixed the indentation.

File: auto_install.sh (hunk 5 above)
Comment (lucianomartin) [reply auto_install.sh:160]: They were missing, I just fixed the indentation.

**Review by mbanth — APPROVED** [review]
Looks good.

File: auto_install.sh (hunk 2 above)
Comment (lucianomartin) [reply auto_install.sh]: I used the latest tags

**Pull request merged**